bigchaindb_driver/driver.py

```diff
@@ -39,6 +40,12 @@ def __init__(self, *, public_key, private_key, node):
         self.public_key = public_key
         self.private_key = private_key
         self.node = node
+        self.transport = transport_class(node)
+
+    def retrieve(self, txid):
+        path = '/transactions' + '/' + txid
+        response = self.transport.forward_request(method='GET', path=path)
+        return response.json()
```

Comment:
I wonder if it'd be useful to wrap the response back in a class, so that certain things are super easy for the user (e.g. `transaction.url`, `transaction.id`, `transaction.json`, `transaction.status` and etc.)